README.md

```diff
@@ -63,7 +63,8 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](https://github.com/lenkon/Portfolio)
+- [Live Demo Link](https://lenkon.github.io/)
+- [Project Presentation Video Link](https://www.loom.com/share/dd0b042569e94d11ac0cb24b3e31e58e)
```

Comment:
[REQUIRED] Your video length should be a maximum of 5 minutes. You didn't need to present the entire code. All you should have presented in your code is the one you found interesting then you move on to the output in the browser.
[REQUIRED] You also forgot to highlight a piece of code you found interesting or proud of making.

Reply:
Hi @fobadara, Thanks for the feedback. I have a doubt. So do I have to highlight only one piece of code or more than one? Can the video length be just about 2 minutes also?

Reply:
Hi @fobadara, I have created a new video that is less than 5 minutes and I have also highlighted a couple of code blocks that I found interesting.